common/repo/repo.proto

```diff
@@ -44,6 +44,9 @@ message File {
   // Empty if referring to a local file
   string commit_id = 5;
   Action action = 6;
+  // Git patch(diff) between current file and the file of the base
```

Comment:
Why do we need this new field?
If the repo is available to the sync job, we can always fetch it from the repo.